doc/data/messages/i/invalid-sequence-index/bad.py

```diff
@@ -0,0 +1,4 @@
+fruit = ['apple', 'banana', 'orange']
+print(fruit[object])  # [invalid-sequence-index]
```

Comment:
```suggestion
fruits = ['apple', 'banana', 'orange']
print(fruits["apple"])  # [invalid-sequence-index]
```
We should use an example without a catch imo.

Reply:
Yes, agree with you.